services/prefect/flows/monitor_failing_scrapers.py

```diff
@@ -1,216 +0,0 @@
-"""Creates a flow to check on scrapers that are consistently failing."""
-from typing import Optional, List
-import datetime
-import dataclasses
-
-import requests
-import prefect
-from prefect import Flow, task
-from prefect.schedules import CronSchedule
-from prefect.tasks.secrets import PrefectSecret
-
```

Comment:
This interacts with the old PrefectV1 graphQL API which has been replaced by a REST API